10-JavaScript/fun-with-functions.md

```diff
@@ -1,34 +1,99 @@
-# Fun with Functions!
+# Intro to Functions!
 
 ## Learning Goals
-- Write and invoke some JavaScript functions
+
+- Intro to syntax (defining and calling), exercise, functions with parameters, exercise
```

Comment:
"exercise" feels weird to be in the learning goals.

Also, I might make "defining", "calling" and "functions with parameters" as sub bullets.

Reply:
lmao this was a note that i leftover for myself, oops, deleting now